earth_enterprise/src/server/wsgi/wms/ogc/common/utils.py

```diff
@@ -137,7 +141,14 @@ def GetServerURL(environ):
   complete_url += urllib.quote(environ.get("REDIRECT_URL", ""))
   complete_url += urllib.quote(environ.get("PATH_INFO", ""))
 
-  return server_url, complete_url
+  if environ.get("HTTP_X_FORWARDED_HOST"):
```

Comment:
What if it is https?

Reply:
I tested both. It still uses this in the header either way.